### PR TITLE
"Speed Up Container Image Builds in Tekton Pipelines" blog post

### PR DESCRIPTION
**Pull request by ggalloro**
This is to add "Speed Up Container Image Builds in Tekton Pipelines" blog post.

Cross post from the article in CDF Blog: https://cd.foundation/blog/2023/10/12/speed-up-container-image-builds-tekton-pipelines/

### CODE REVIEW

**Review by afrittoli — APPROVED** [review]
Thanks @ggalloro, this looks good to me!
I appreciate you taking the time to post on the Tekton blog.

I wonder whether it would make sense to add a note at the end like "originally posted @ <link>"

**Comment by afrittoli** [comment]
Looking for a second review @AlanGreene @vdemeester

**Comment by AlanGreene** [comment]
Thanks @ggalloro. I agree with Andrea's comment, this should include a link to the original post.

Also looks like this relies on content from https://github.com/tektoncd/pipeline/pull/7312 being available. Since it points at the `main` branch rather than a particular commit / tag there's a risk that we'll end up with dead links in the blog post in future.

Would it make sense to include those files in the website repo alongside the blog post instead?

/hold

**Comment by ggalloro** [comment]
Thanks @AlanGreene will try to do that.

Any advice to which path to put the sample code in ?

**Comment by AlanGreene** [comment]
A sub-folder of the blog post location would make sense to keep it all together

**Comment by ggalloro** [comment]
Added a commit to this PR w example code under the blog folder.

Changed the links to example code in the post  with absolute links as https://github.com/tektoncd/website/tree/main/content/en/blog/2023/speeding-up-container-image-builds-in-tekton-pipelines/tekton-speed-builds/

Added note with link to original post in CDF blog

**Comment by ggalloro** [comment]
Hi @AlanGreene @afrittoli is there any further action needed from my side to get this PR reviewed ?
Many thanks !

File: content/en/blog/2023/speeding-up-container-image-builds-in-tekton-pipelines/index.md
Comment (AlanGreene) [inline content/en/blog/2023/speeding-up-container-image-builds-in-tekton-pipelines/index.md:4]: The date isn't showing up properly in the preview, I think it requires zero-padding the date

image
image

```suggestion
date: 2023-11-02
```

**Review by AlanGreene — CHANGES_REQUESTED** [review]
Apologies for the delay @ggalloro, I missed the notification.

Content looks good, just a small issue with the date and hopefully then it's good to go 🤞

**Review by AlanGreene — APPROVED** [review]
Thanks @ggalloro 
/lgtm

**Comment by afrittoli** [comment]
🎉

**Pull request merged**
